hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -425,6 +425,25 @@ public FileChecksum getFileChecksum() {
     return fileChecksum;
   }
 
+  @Override
+  public String toString() {
+    return "OmKeyInfo{" +
+        "volumeName='" + volumeName + '\'' +
+        ", bucketName='" + bucketName + '\'' +
+        ", keyName='" + keyName + '\'' +
+        ", dataSize=" + dataSize +
+        ", keyLocationVersions=" + keyLocationVersions +
+        ", creationTime=" + creationTime +
+        ", modificationTime=" + modificationTime +
+        ", replicationConfig=" + replicationConfig +
+        ", encInfo=" + encInfo +
```

Comment:
Encryption info seems sensitive to be logged, please check and remove this.

Reply:
Good catch. I have changed it into logging `null` or `<REDACTED>`:

```
", encInfo=" + (encInfo == null ? "null" : "<REDACTED>") +
```

Reply:
Done in 3be3d149431e04309cdd341685df51605ce5a8b0